scripts/changeset/get-full-changelog.mts

```diff
@@ -144,9 +157,10 @@ export async function getFullChangelog(octokit: Octokit) {
   const breaking = listBreakingMd(changelogs);
   const nonBreaking = listNonBreakingMd(changelogs);
 
-  let content = `# RELEASE - ${process.env.RELEASE_TAG ?? 'TBD'}\n\n`;
-  content += breaking ? `# Breaking\n\n${breaking}` : '';
-  content += breaking && nonBreaking && '\n\n---\n\n';
+  let content = ``;
```

Comment:
It became obsolete after the version was [added to the title](https://github.com/FuelLabs/fuels-ts/pull/1972/files#r1543512598).